packages/engine/lib/execution/src/package/comms.rs

```diff
@@ -70,12 +70,12 @@ impl<C: Comms> PackageComms<C> {
             .await
     }
 
-    #[allow(dead_code)]
+    // TODO: UNUSED: triage
```

Comment:
Why did you add these back? This doesn't need a triage, we've decided to keep it (we should add a comment saying why it's unused at the moment as _the current_ packages don't need to add/remove agents)

Reply:
I added a description why they are unused and also added a doc-string: f1067a787f4e9440738fea6e8fdf080a3966922c